shiny/www/shared/bootstrap/_version.json

```diff
@@ -1,7 +1,7 @@
 {
   "note!": "This file is auto-generated by scripts/htmlDependencies.R",
-  "shiny_version": "Github (rstudio/shiny@54988c17c8fae0e74e69b3b7fd99edaa435dc3c6)",
-  "bslib_version": "Github (rstudio/bslib@7bc2171f0311fd980962e9a22571e30b3f77f2fa)",
-  "htmltools_version": "CRAN (R 4.2.0)",
+  "shiny_version": "Github (rstudio/shiny@f71f1256b8ed1ee133c5aea281bbc750a379460c)",
```

Comment:
Why has the shiny version changed but shiny assets haven't?

Reply:
I think that when I ran the update script previously, I did chose not to stage and commit the changes in this file because I just wanted to update shiny, but I did not want to touch bslib. I didn't realize that I would have to commit this line in this file.

This process makes it difficult to just update shiny without updating bslib. How should one do that?

Reply:
I'm not sure but I'm okay with what we've done in this and the other PR (not updating package version json files unless we update those packages). I made a similar decision about a version snapshot for htmltools.

Reply:
To be clear, in my case the htmltools snapshot changed the R version from 4.2 to 4.3, so it's not like there were any dragons hiding there.

Reply:
I think, for now, probably the best way would be to carefully to bring the relevant SHA into the `pkg_install()` command (from `version.json`), but probably not commit that change in `scripts/htmlDependencies.R`

Reply:
Yeah that's very much what I did, I just created a branch in bslib to store that branch and its commits. I'd rather use the branch than the specific commit because it's clearer (if you go to the bslib repo) which commits were included exactly, especially since it's a branch of cherry-picked commits rather than a point in time on bslib's main branch.

Reply:
Oh except that I think we _should_  commit the install command we used for the last import.